.prettierignore

```diff
@@ -1,7 +1,15 @@
 *.d.ts
+.codesandbox
+.git
+.github
+.storybook
 bower_components
 build
-clay-css
+packages/clay-css/src/scss/bootstrap/**/**/**/*.scss
+packages/clay-css/**/**/**/**/**/*.js
+packages/clay-css/**/**/**/**/**/*.md
+packages/clay-css/**/**/**/**/**/*.json
```

Comment:
Also are we able to format the js md and json in clay? Or are those ignored for a specific reason?

Reply:
The test site uses several third-party js plugins. I didn't want to ignore them separately since the test site isn't a critical part of the Clay repo. For example, we shouldn't be formatting `svg4everybody.js`, `clay-css/src/js/site/jquery.ba-outside-events.min.js`, `sassdoc-theme-clay-css/dist/index.js`, `sassdoc-theme-clay-css/dist/nunjucks.js`, etc.

If it's something you want I can add it.  I was thinking along the lines of why process files that don't really matter?

Reply:
You don't need all those repeated `**` — one is equivalent to any number of intervening directories (full syntax [described here](https://git-scm.com/docs/gitignore#_pattern_format)).

For example:

- `packages/clay-css/**/*.js` is the same as `packages/clay-css/**/**/**/**/**/*.js`

If you really want to say "match *exactly* 5 levels down" you would use a single `*` at each level:

- `packages/clay-css/*/*/*/*/*/*.js`

If you want to ignore dist-like third-party stuff like the stuff you list you can either list it explicitly:

```
/packages/clay-css/dist
/packages/clay-css/src/js/site/jquery.*
# etc
```

or ignore everything then whitelist specific stuff:

```
/packages/clay-css/**/*.js
!packages/clay-css/specific/place/to/whitelist/**/*.js
```

Reply:
Thanks @wincent, I'll make these changes and push it up.